[2024-04-22 17:15:31.267] [PSS] [info] ***********  Log Begin **********
[2024-04-22 17:15:31.268] [PSS] [info] process id= 18076
[2024-04-22 17:15:31.268] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x201894ed550, this= 0x20298c58950
[2024-04-22 17:15:31.268] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= Undefined, this= 0x20298c58950
[2024-04-22 17:15:31.268] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x20298c58950
[2024-04-22 17:15:31.268] [PSS] [error] [18076|6312]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x20298c58950
[2024-04-22 17:15:31.268] [PSS] [debug] [18076|6312]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x20298c58950
[2024-04-22 17:15:31.268] [PSS] [debug] [18076|6312]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x2028cb5b070
[2024-04-22 17:15:31.268] [PSS] [info] [18076|6312]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x2028cb5b070
[2024-04-22 17:15:31.268] [PSS] [debug] [18076|6312]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x2028cb5b080, this= 0x2017ff1ccd0
[2024-04-22 17:15:33.307] [PSS] [error] [18076|24724]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x2028cb5b070
[2024-04-22 17:15:33.307] [PSS] [debug] [18076|24724]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x20298c58950
[2024-04-22 17:15:33.307] [PSS] [error] [18076|24724]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x20298c58950
[2024-04-22 17:24:38.927] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x2028c5bf9e0, this= 0x20187950b50
[2024-04-22 17:24:38.927] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x20298c58950
[2024-04-22 17:24:38.927] [PSS] [debug] [18076|304]transport_client_tcp2.cpp:71 Stop: called, this= 0x2028cb5b070
[2024-04-22 17:24:38.928] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x20187950b50
[2024-04-22 17:24:38.928] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x20187950b50
[2024-04-22 17:24:38.928] [PSS] [error] [18076|6624]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x20187950b50
[2024-04-22 17:24:38.928] [PSS] [debug] [18076|6624]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x20187950b50
[2024-04-22 17:24:38.928] [PSS] [debug] [18076|6624]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x2028cb5bf20
[2024-04-22 17:24:38.928] [PSS] [info] [18076|6624]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x2028cb5bf20
[2024-04-22 17:24:38.928] [PSS] [debug] [18076|6624]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x2028cb5bf30, this= 0x2018af71da0
[2024-04-22 17:24:40.953] [PSS] [error] [18076|23992]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x2028cb5bf20
[2024-04-22 17:24:40.953] [PSS] [debug] [18076|23992]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x20187950b50
[2024-04-22 17:24:40.953] [PSS] [error] [18076|23992]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x20187950b50
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x20182c9c890, this= 0x2018cece060
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x20187950b50
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|304]transport_client_tcp2.cpp:71 Stop: called, this= 0x2028cb5bf20
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x2018cece060
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x2018cece060
[2024-04-22 17:33:11.846] [PSS] [error] [18076|13468]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x2018cece060
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|13468]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x2018cece060
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|13468]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x201ce4ecc00
[2024-04-22 17:33:11.846] [PSS] [info] [18076|13468]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x201ce4ecc00
[2024-04-22 17:33:11.846] [PSS] [debug] [18076|13468]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x201ce4ecc10, this= 0x20377845a30
[2024-04-22 17:33:13.886] [PSS] [error] [18076|17136]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x201ce4ecc00
[2024-04-22 17:33:13.886] [PSS] [debug] [18076|17136]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x2018cece060
[2024-04-22 17:33:13.886] [PSS] [error] [18076|17136]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x2018cece060
[2024-04-22 17:40:48.329] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x201d10c04d0, this= 0x201d097d490
[2024-04-22 17:40:48.329] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x2018cece060
[2024-04-22 17:40:48.329] [PSS] [debug] [18076|304]transport_client_tcp2.cpp:71 Stop: called, this= 0x201ce4ecc00
[2024-04-22 17:40:48.330] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x201d097d490
[2024-04-22 17:40:48.330] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x201d097d490
[2024-04-22 17:40:48.330] [PSS] [error] [18076|1240]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x201d097d490
[2024-04-22 17:40:48.330] [PSS] [debug] [18076|1240]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x201d097d490
[2024-04-22 17:40:48.330] [PSS] [debug] [18076|1240]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x201ce4ec040
[2024-04-22 17:40:48.330] [PSS] [info] [18076|1240]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x201ce4ec040
[2024-04-22 17:40:48.330] [PSS] [debug] [18076|1240]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x201ce4ec050, this= 0x2018af6fdb0
[2024-04-22 17:40:50.372] [PSS] [error] [18076|10460]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x201ce4ec040
[2024-04-22 17:40:50.372] [PSS] [debug] [18076|10460]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x201d097d490
[2024-04-22 17:40:50.372] [PSS] [error] [18076|10460]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x201d097d490
[2024-04-22 17:41:41.424] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x202d0c7b980, this= 0x2018cece060
[2024-04-22 17:41:41.424] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x201d097d490
[2024-04-22 17:41:41.424] [PSS] [debug] [18076|304]transport_client_tcp2.cpp:71 Stop: called, this= 0x201ce4ec040
[2024-04-22 17:41:41.425] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x2018cece060
[2024-04-22 17:41:41.425] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x2018cece060
[2024-04-22 17:41:41.425] [PSS] [error] [18076|5176]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x2018cece060
[2024-04-22 17:41:41.425] [PSS] [debug] [18076|5176]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x2018cece060
[2024-04-22 17:41:41.425] [PSS] [debug] [18076|5176]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x201ce4ebd50
[2024-04-22 17:41:41.425] [PSS] [info] [18076|5176]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x201ce4ebd50
[2024-04-22 17:41:41.425] [PSS] [debug] [18076|5176]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x201ce4ebd60, this= 0x202dcf2fdd0
[2024-04-22 17:41:43.471] [PSS] [error] [18076|19312]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x201ce4ebd50
[2024-04-22 17:41:43.471] [PSS] [debug] [18076|19312]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x2018cece060
[2024-04-22 17:41:43.471] [PSS] [error] [18076|19312]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x2018cece060
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x202dbba3640, this= 0x201d097d490
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x2018cece060
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|304]transport_client_tcp2.cpp:71 Stop: called, this= 0x201ce4ebd50
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x201d097d490
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x201d097d490
[2024-04-22 17:50:05.669] [PSS] [error] [18076|7084]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x201d097d490
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|7084]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x201d097d490
[2024-04-22 17:50:05.669] [PSS] [debug] [18076|7084]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x201ce4ed1e0
[2024-04-22 17:50:05.670] [PSS] [info] [18076|7084]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x201ce4ed1e0
[2024-04-22 17:50:05.670] [PSS] [debug] [18076|7084]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x201ce4ed1f0, this= 0x202a083da20
[2024-04-22 17:50:07.707] [PSS] [error] [18076|9276]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x201ce4ed1e0
[2024-04-22 17:50:07.707] [PSS] [debug] [18076|9276]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x201d097d490
[2024-04-22 17:50:07.707] [PSS] [error] [18076|9276]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x201d097d490
[2024-04-22 17:51:15.459] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x2028ce22f00, this= 0x2037699fda0
[2024-04-22 17:51:15.459] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x201d097d490
[2024-04-22 17:51:15.459] [PSS] [debug] [18076|304]transport_client_tcp2.cpp:71 Stop: called, this= 0x201ce4ed1e0
[2024-04-22 17:51:15.460] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x2037699fda0
[2024-04-22 17:51:15.460] [PSS] [debug] [18076|304]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x2037699fda0
[2024-04-22 17:51:15.460] [PSS] [error] [18076|20232]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x2037699fda0
[2024-04-22 17:51:15.460] [PSS] [debug] [18076|20232]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x2037699fda0
[2024-04-22 17:51:15.460] [PSS] [debug] [18076|20232]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x201d1f9b680
[2024-04-22 17:51:15.460] [PSS] [info] [18076|20232]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x201d1f9b680
[2024-04-22 17:51:15.460] [PSS] [debug] [18076|20232]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x201d1f9b690, this= 0x202dcf303f0
[2024-04-22 17:51:17.503] [PSS] [error] [18076|5756]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x201d1f9b680
[2024-04-22 17:51:17.503] [PSS] [debug] [18076|5756]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x2037699fda0
[2024-04-22 17:51:17.503] [PSS] [error] [18076|5756]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x2037699fda0
[2024-04-22 21:38:10.527] [PSS] [info] ***********  Log Begin **********
[2024-04-22 21:38:10.527] [PSS] [info] process id= 18576
[2024-04-22 21:38:10.527] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ec102d2b40, this= 0x1ed7e75bf80
[2024-04-22 21:38:10.527] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= Undefined, this= 0x1ed7e75bf80
[2024-04-22 21:38:10.527] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed7e75bf80
[2024-04-22 21:38:10.527] [PSS] [error] [18576|3952]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed7e75bf80
[2024-04-22 21:38:10.527] [PSS] [debug] [18576|3952]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed7e75bf80
[2024-04-22 21:38:10.527] [PSS] [debug] [18576|3952]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ec5dc8efd0
[2024-04-22 21:38:10.527] [PSS] [info] [18576|3952]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ec5dc8efd0
[2024-04-22 21:38:10.527] [PSS] [debug] [18576|3952]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ec5dc8efe0, this= 0x1ed325f70e0
[2024-04-22 21:38:12.562] [PSS] [error] [18576|7288]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ec5dc8efd0
[2024-04-22 21:38:12.562] [PSS] [debug] [18576|7288]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed7e75bf80
[2024-04-22 21:38:12.562] [PSS] [error] [18576|7288]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed7e75bf80
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ec14ed1ca0, this= 0x1ed3a85c020
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ed7e75bf80
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ec5dc8efd0
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ed3a85c020
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed3a85c020
[2024-04-22 21:38:30.833] [PSS] [error] [18576|15756]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed3a85c020
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|15756]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed3a85c020
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|15756]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ed7f1590a0
[2024-04-22 21:38:30.833] [PSS] [info] [18576|15756]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ed7f1590a0
[2024-04-22 21:38:30.833] [PSS] [debug] [18576|15756]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ed7f1590b0, this= 0x1ed2ec2c2d0
[2024-04-22 21:38:32.849] [PSS] [error] [18576|7500]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ed7f1590a0
[2024-04-22 21:38:32.850] [PSS] [debug] [18576|7500]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed3a85c020
[2024-04-22 21:38:32.850] [PSS] [error] [18576|7500]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed3a85c020
[2024-04-22 21:46:50.524] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ec11771160, this= 0x1ed2f0bf060
[2024-04-22 21:46:50.524] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ed3a85c020
[2024-04-22 21:46:50.524] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ed7f1590a0
[2024-04-22 21:46:50.525] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ed2f0bf060
[2024-04-22 21:46:50.525] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed2f0bf060
[2024-04-22 21:46:50.525] [PSS] [error] [18576|4764]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed2f0bf060
[2024-04-22 21:46:50.525] [PSS] [debug] [18576|4764]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed2f0bf060
[2024-04-22 21:46:50.525] [PSS] [debug] [18576|4764]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ed7f15a820
[2024-04-22 21:46:50.525] [PSS] [info] [18576|4764]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ed7f15a820
[2024-04-22 21:46:50.525] [PSS] [debug] [18576|4764]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ed7f15a830, this= 0x1ec0eea2f70
[2024-04-22 21:46:52.561] [PSS] [error] [18576|24836]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ed7f15a820
[2024-04-22 21:46:52.561] [PSS] [debug] [18576|24836]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed2f0bf060
[2024-04-22 21:46:52.561] [PSS] [error] [18576|24836]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed2f0bf060
[2024-04-22 23:43:55.198] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ecfa79fc60, this= 0x1ed3ee0e790
[2024-04-22 23:43:55.198] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ed2f0bf060
[2024-04-22 23:43:55.198] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ed7f15a820
[2024-04-22 23:43:55.202] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ed3ee0e790
[2024-04-22 23:43:55.202] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed3ee0e790
[2024-04-22 23:43:55.202] [PSS] [error] [18576|22480]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed3ee0e790
[2024-04-22 23:43:55.203] [PSS] [debug] [18576|22480]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed3ee0e790
[2024-04-22 23:43:55.203] [PSS] [debug] [18576|22480]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ed7f15a530
[2024-04-22 23:43:55.203] [PSS] [info] [18576|22480]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ed7f15a530
[2024-04-22 23:43:55.203] [PSS] [debug] [18576|22480]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ed7f15a540, this= 0x1ec0e947670
[2024-04-22 23:43:57.239] [PSS] [error] [18576|28124]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ed7f15a530
[2024-04-22 23:43:57.239] [PSS] [debug] [18576|28124]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed3ee0e790
[2024-04-22 23:43:57.239] [PSS] [error] [18576|28124]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed3ee0e790
[2024-04-22 23:51:22.142] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ec14ed6110, this= 0x1ed49e52b70
[2024-04-22 23:51:22.142] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ed3ee0e790
[2024-04-22 23:51:22.142] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ed7f15a530
[2024-04-22 23:51:22.142] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ed49e52b70
[2024-04-22 23:51:22.142] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed49e52b70
[2024-04-22 23:51:22.142] [PSS] [error] [18576|25180]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed49e52b70
[2024-04-22 23:51:22.154] [PSS] [debug] [18576|25180]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed49e52b70
[2024-04-22 23:51:22.154] [PSS] [debug] [18576|25180]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ec10505250
[2024-04-22 23:51:22.154] [PSS] [info] [18576|25180]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ec10505250
[2024-04-22 23:51:22.154] [PSS] [debug] [18576|25180]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ec10505260, this= 0x1ed4a89f4f0
[2024-04-22 23:51:24.199] [PSS] [error] [18576|19784]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ec10505250
[2024-04-22 23:51:24.199] [PSS] [debug] [18576|19784]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed49e52b70
[2024-04-22 23:51:24.199] [PSS] [error] [18576|19784]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed49e52b70
[2024-04-23 00:04:09.075] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ed2f37a340, this= 0x1ed3ee0e790
[2024-04-23 00:04:09.075] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ed49e52b70
[2024-04-23 00:04:09.077] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ec10505250
[2024-04-23 00:04:09.077] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ed3ee0e790
[2024-04-23 00:04:09.077] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed3ee0e790
[2024-04-23 00:04:09.077] [PSS] [error] [18576|3128]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed3ee0e790
[2024-04-23 00:04:09.077] [PSS] [debug] [18576|3128]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed3ee0e790
[2024-04-23 00:04:09.077] [PSS] [debug] [18576|3128]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ec10505830
[2024-04-23 00:04:09.077] [PSS] [info] [18576|3128]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ec10505830
[2024-04-23 00:04:09.077] [PSS] [debug] [18576|3128]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ec10505840, this= 0x1ed328a3260
[2024-04-23 00:04:11.107] [PSS] [error] [18576|4344]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ec10505830
[2024-04-23 00:04:11.107] [PSS] [debug] [18576|4344]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed3ee0e790
[2024-04-23 00:04:11.107] [PSS] [error] [18576|4344]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed3ee0e790
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ec14ecb3f0, this= 0x1ec15da3030
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ed3ee0e790
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ec10505830
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ec15da3030
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ec15da3030
[2024-04-23 00:18:45.817] [PSS] [error] [18576|13428]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ec15da3030
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|13428]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ec15da3030
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|13428]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ec10505250
[2024-04-23 00:18:45.817] [PSS] [info] [18576|13428]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ec10505250
[2024-04-23 00:18:45.817] [PSS] [debug] [18576|13428]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ec10505260, this= 0x1ed4a8a0c20
[2024-04-23 00:18:47.862] [PSS] [error] [18576|6108]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ec10505250
[2024-04-23 00:18:47.862] [PSS] [debug] [18576|6108]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ec15da3030
[2024-04-23 00:18:47.862] [PSS] [error] [18576|6108]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ec15da3030
[2024-04-23 01:29:55.502] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:25 StreamingDriverInterfaceImpl: callback= 0x1ed75829050, this= 0x1ed3ee0e790
[2024-04-23 01:29:55.503] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:30 ~StreamingDriverInterfaceImpl: called, this= 0x1ec15da3030
[2024-04-23 01:29:55.503] [PSS] [debug] [18576|29240]transport_client_tcp2.cpp:71 Stop: called, this= 0x1ec10505250
[2024-04-23 01:29:55.504] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:95 SetTrackingOrigin: tracking origin= StageLevel, this= 0x1ed3ee0e790
[2024-04-23 01:29:55.504] [PSS] [debug] [18576|29240]streaming_driver_interface_impl.cpp:39 ConnectToService: try to connect to service, this= 0x1ed3ee0e790
[2024-04-23 01:29:55.505] [PSS] [error] [18576|25500]streaming_driver_interface_impl.cpp:572 InvokeSetTrackingOrigin: "invalid media session", !media_session_, this= 0x1ed3ee0e790
[2024-04-23 01:29:55.505] [PSS] [debug] [18576|25500]streaming_driver_interface_impl.cpp:469 InvokeConnectToService: connect to service through the network, port= 54000, this= 0x1ed3ee0e790
[2024-04-23 01:29:55.505] [PSS] [debug] [18576|25500]transport_client_tcp2.cpp:34 Start: buffer size(send\recv): 1048576\1048576, this= 0x1ec10504690
[2024-04-23 01:29:55.505] [PSS] [info] [18576|25500]transport_client_tcp2.cpp:58 Start: server= 127.0.0.1:54000, local=0.0.0.0:0, this= 0x1ec10504690
[2024-04-23 01:29:55.505] [PSS] [debug] [18576|25500]transport_data_receiver_raw.cpp:34 UpdateConfig: enable_channel= 1, enable_packet= 1, sink= 0x1ec105046a0, this= 0x1e9ceed26e0
[2024-04-23 01:29:57.554] [PSS] [error] [18576|28460]transport_client_tcp2.cpp:106 HandleConnect: connected error= 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다, this= 0x1ec10504690
[2024-04-23 01:29:57.554] [PSS] [debug] [18576|28460]streaming_driver_interface_impl.cpp:248 NotifyConnectResult: network connect to service result= 10061, this= 0x1ed3ee0e790
[2024-04-23 01:29:57.554] [PSS] [error] [18576|28460]streaming_driver_interface_impl.cpp:250 NotifyConnectResult: connect to service network error= 10061, this= 0x1ed3ee0e790
